File: C++Way/100Demos/Swap.java
public class Swap {
    public static void main(String[] args) {
        int a = 11;
        int b = 22;
        new Swap().swap(a, b);
        System.out.printf("a=%d, b=%d\n", a, b);

        int temp = 0;
        temp = a;
        a = b;
        b = temp;
        System.out.printf("a=%d, b=%d", a, b);
    }
    
    public void swap(int a, int b) {
        int temp = a;
        a = b;
        b = temp;
    }
}

File: CLionProject/ClionCode/Swap.cpp
//
// Created by yangw on 2020/10/29.
//

#include<iostream>
#include<string>

using namespace std;

int main(){
    int a, b;
    int temp = 0;

    cout<<"请输入两个数字， 以空格隔开:"<<endl;
    cin>>a;
    cin>>b;
    cout<<"a = "<<a<<", b = "<<b<<endl;

    temp = a;
    a = b;
    b = temp;
    cout<<"a = "<<a<<", b = "<<b<<endl;
}